test/e2e/testing-manifests/storage-csi/gce-pd/node_ds.yaml

```diff
@@ -60,7 +60,7 @@ spec:
       volumes:
         - name: registration-dir
           hostPath:
-            path: /var/lib/kubelet/plugins/
+            path: /var/lib/kubelet/plugins_registry/
             type: Directory
```

Comment:
I am seeing:
```
MountVolume.SetUp failed for volume "registration-dir" : hostPath type check failed: /var/lib/kubelet/plugins_registry/ is not a directory
```

To fix this, change this `type` from `Directory` to `HostPathDirectoryOrCreate`.